ui/raidboss/data/07-dt/raid/r6n.ts

```diff
@@ -1,14 +1,346 @@
+    'location': '16',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '17': {
+    'location': '17',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '18': {
+    'location': '18',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '19': {
+    'location': '19',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1A': {
+    'location': '1A',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1B': {
+    'location': '1B',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1C': {
+    'location': '1C',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1D': {
+    'location': '1D',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1E': {
+    'location': '1E',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+  },
+  '1F': {
+    'location': '1F',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+    'flags3': '00200010',
+  },
+  '20': {
+    'location': '20',
+    'flags0': '00040004',
+  },
+  '21': {
+    'location': '21',
+    'flags0': '00040004',
+  },
+  '22': {
+    'location': '22',
+    'flags0': '00040004',
+  },
+  '23': {
+    'location': '23',
+    'flags0': '00020001',
+    'flags1': '00040004',
+    'clear2': '00080004',
+    'flags3': '00200010',
+  },
+
+  // River phase rain effect
+  '24': {
+    'location': '24',
+    // Rain on
+    'flags0': '00020001',
+    // Rain off
+    'clear1': '00080004',
+  },
+
+  // Unknown, something to do with floor changing maybe?
+  '25': {
+    'location': '25',
+    'flags0': '00020004',
+  },
+} as const;
+console.assert(mapEffectData);
+
+// Arrows, bombs, etc get their position set before the animation for the tether starts showing.
+// Probably we want a general `ActorSetPos` tracking trigger for this fight.
+
+const headMarkerData = {
+  // Spread marker from Mousse Touch-up
+  'spreadMarker': '008B',
+  // 5-hit stack marker from Pudding Party
+  'stack': '0131',
+  // Tankbuster circles from Color Riot
+  'tankbuster': '0156',
+  // Fire stacks on healers
+  'fireStack': '024C',
+  // Lightning spread marker
+  'lightningSpread': '024D',
+} as const;
+console.assert(headMarkerData);
+
 const triggerSet: TriggerSet<Data> = {
   id: 'AacCruiserweightM2',
   zoneId: ZoneId.AacCruiserweightM2,
   timelineFile: 'r6n.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R6N Mousse Mural',
+      type: 'StartsUsing',
+      netRegex: { id: 'A66F', source: 'Sugar Riot', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R6N Pudding Party',
+      type: 'StartsUsing',
+      netRegex: { id: 'A66D', source: 'Sugar Riot', capture: true },
+      response: Responses.stackMarkerOn(),
```

Comment:
This is self-targeted on the boss, we'll have to use the headmarker to call the actual stack target.